testutil/singlevm/setup.sh

```diff
@@ -105,6 +105,127 @@ function ctrl_c() {
     exit 1
 }
 
+function createWorkloads() {
+	mkdir -p ${ciao_bin}/workload_examples
+	if [ ! -d ${ciao_bin}/workload_examples ]
+	then
+		echo "FATAL ERROR: Unable to create ${ciao_bin}/workload_examples"
+		exit 1
+
+	fi
+
+	cwd=`pwd`
+
+	sudo chmod 755 "$ciao_bin"/workload_examples
+
+	# create a VM test workload with ssh capability
+	echo "Generating VM cloud-init"
+	(
+	cat <<-EOF
+	---
+	#cloud-config
+	users:
+	  - name: demouser
+	    gecos: CIAO Demo User
+	    lock-passwd: false
+	    passwd: ${test_passwd}
+	    sudo: ALL=(ALL) NOPASSWD:ALL
+	    ssh-authorized-keys:
+	      - ${test_sshkey}
+	...
+	EOF
+	) > "$ciao_bin"/workload_examples/vm-test.yaml
+
+	# Get the image ID for the fedora cloud image
+	id=$(ciao-cli image list -f='{{$x := filter . "Name" "Fedora Cloud Base 24-1.2"}}{{if gt (len $x) 0}}{{(index $x 0).ID}}{{end}}')
+
+	# add 2 vm test workloads
+	echo "Generating Fedora VM test workload"
+	(
+	cat <<-EOF
+	description: "Fedora test VM"
+	vm_type: qemu
+	fw_type: legacy
+	image_id: "$id"
+	defaults:
+	    vcpus: 2
+	    mem_mb: 512
```

Comment:
I don't think these values match what we used to have in SingleVM.  If I read the resource file correctly, the fedora workload had 128MB or ram and 80GB.  These differences might be significant for ciao-down users.